### PR TITLE
Encapsulated code into functions

### PR DESCRIPTION
**Pull request by sandrinesoeharjono**
Code for:
1. calculating most values (V0, H0, R0, mortality, etc) 
2. plotting these values
is now encapsulated in official reusable functions in Functions.jl (to ensure that the values and plots are generated in the exact same way for each of the 3 main sections. The output path for each plot is also called as a parameter, in order to immediately write it as a PNG upon creation.

Output directories for each figure are also set into a variable in the respective "figxparams.jl" scripts.

### CODE REVIEW

File: Model_scripts/Functions.jl [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
-:param indiv: vector of number of individuals of each group
-    (first idx is uninfected hosts; 2:end to each strain)
-:param range: window for each time step
-:param parameters: model parameters
-:returns: differential eqn to solve
-"""
+
+# Statistical functions ######################################################
 function comp_densities(indiv, range, parameters)
+    """
```
Comment (sandrinesoeharjono) [inline Model_scripts/Functions.jl]: @kiristern i may be wrong but I believe you were the one who took these code blocks outside of the functions. they should remain inside, as to inform what the function does :)

**Review by sandrinesoeharjono — COMMENTED** [review]
if that works, we can do the same for the supplemental scripts that were recently added

File: Model_scripts/Functions.jl (hunk 1 above)
Comment (kiristern) [reply Model_scripts/Functions.jl]: i was following the Documenter.jl format and it says to put the code blocks before the function. i'm okay with moving them back inside

Comment (tpoisot) [reply Model_scripts/Functions.jl]: they should definitely be outside!

**Review by tpoisot — APPROVED** [review]
keep the docstrings outside and it's all good

**Comment by kiristern** [comment]
@tpoisot did the code run when you tried? I am getting errors

**Comment by tpoisot** [comment]
I haven't run it yet - which version of Julia are you using?

**Comment by kiristern** [comment]
1.6.1. We resolved the issue and it seems to work now

**Review by kiristern — APPROVED** [review]

File: Model_scripts/Functions.jl (hunk 1 above)
Comment (sandrinesoeharjono) [reply Model_scripts/Functions.jl]: yup i see now! my bad, i got too used to the pythonic way of doing things. you guys are right, thanks for clarifying!

**Pull request merged**